au/code/au/math_test.cc

```diff
@@ -70,6 +71,36 @@ TEST(abs, SameAsStdAbsForNumericTypes) {
     EXPECT_EQ(abs(1), 1);
 }
 
+TEST(cbrt, OutputRepDependsOnInputRep) {
+    EXPECT_THAT(cbrt(cubed(meters)(8)), QuantityEquivalent(meters(2.)));
+    EXPECT_THAT(cbrt(cubed(meters)(8.)), QuantityEquivalent(meters(2.)));
+    EXPECT_THAT(cbrt(cubed(meters)(8.f)), QuantityEquivalent(meters(2.f)));
+    EXPECT_THAT(cbrt(cubed(meters)(8.L)), QuantityEquivalent(meters(2.L)));
+}
+
+TEST(cbrt, SameAsStdCbrtForNumericTypes) {
+    EXPECT_EQ(cbrt(1), std::cbrt(1));
+    EXPECT_EQ(cbrt(1.), std::cbrt(1.));
+    EXPECT_EQ(cbrt(1.f), std::cbrt(1.f));
+    EXPECT_EQ(cbrt(1.L), std::cbrt(1.L));
+}
+
+TEST(cbrt, CanConvertIfConversionFactorRational) {
+    const auto geo_mean_length = cbrt(inches(1) * meters(1) * yards(1));
+
+    // Using Quantity-equivalent Unit just retrieves the stored value.
+    const auto retrieved_value = geo_mean_length.in(cbrt(inch * meter * yards));
+    EXPECT_THAT(retrieved_value, SameTypeAndValue(1.0));
```

Comment:
If it's just retrieving the stored value, does the value really come out as a `double`?

Reply:
I noticed that too (this test is a copy-and-modify of the corresponding `sqrt` one).  Then I realized that it's getting passed through `std::cbrt`, which is why it becomes a `double`.

The comment is unclear, though.  At first glance, it makes it seem like the "stored" value should be `1 * 1 * 1`, but we're actually referring to the value stored in the result of `cbrt` (i.e., `geo_mean_length`).  I'll update all the relevant comments.